sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java

```diff
@@ -389,16 +391,26 @@ public void testGlobalCombineWithDefaultsAndTriggers() {
         .apply(Sum.integersGlobally())
         .apply(ParDo.of(new FormatPaneInfo()));
 
-    PAssert.that(output).containsInAnyOrder("1: false", "2: true");
+    // The elements produced are nondeterministic, but it should certainly have
```

Comment:
Emphasize that whether or not we *actually* get the partial result is non deterministic. At first I thought you just meant the order, so I was going to suggest the in any order again ;)